generators/app/templates/common/README.md

```diff
@@ -238,11 +238,40 @@ If the plan suits what you were expecting, provision the development environment
   terraform apply
   ```
 
+<% } -%>
+<% if (includeApi && includeGCPInfrastructureCodeForApi) { -%>
+## Infrastructure
+
+The infrastructure of the project consists of GCP resources, provisioned with Terraform.
+The Terraform code for all resources can be found in the `terraform` folder.
+
+### Process to add/delete/update resources
+
+Select the environment you want to provision:
+```bash
+terraform workspace select <env_name>
+```
+
+Then check the module adding/deletion plan
+  ```bash
+  terraform plan
+  ```
+
+If the plan suits what you were expecting, provision the development environment by running:
+  ```bash
+  terraform apply
+  ```
+
+### Access the API
```

Comment:
Do you know the pricing of the infra? Could be nice to include it in README and in prompting, as we do for AWS.